### ISSUE TITLE
Issues pt1

### ISSUE THREAD

**Issue by panaut0lordv** [jaredrybarski/Zad2#1]
AWS keys in a repo? use profiles (or at least env vars)
`terraform fmt` is your friend
Polish keywords? Brzmisz jak polak
Public SSH? Seriously?

**Comment by jaredrybarski** [comment]
thanks for comment, 
it's my first tf code ever so every hint is very valuable. Of course, I removed the keys before putting the code on git. Env vars will be next. 
Thanks

**Comment by panaut0lordv** [comment]
Yup, tested the keys back when I commented so can confirm they were not usable. Disabling them is one thing, another is they shouldn't be there in the first place, doesn't matter whether it's PoC/sandbox grade code or production deployment.

Cheers!

**Comment by jaredrybarski** [comment]
I will keep it in mind

Thanks !!!
